src/client/state/state.js

```diff
@@ -1,5 +1,4 @@
 import { createContext } from 'react';
 
-const authContext = createContext();
+export const authContext = createContext();
```

Comment:
since this will be used as a component, it should be capitalized (`AuthContext`)